test/documentation/CMakeLists.txt

```diff
@@ -10,14 +10,13 @@ cmake_minimum_required(VERSION 3.7)
 
 ### Find seqan3
 include (../../build_system/seqan3-config-version.cmake)
-set (SEQAN3_VERSION "${PACKAGE_VERSION}")
```

Comment:
this simulates `find_package(seqan3)` which we explicitly don't do for documentation, because we don't need any C++ compiler.

Reply:
I moved it to the version config, because I'd like to use `3.0.3-rc2` in the documentation, but it is not a valid version for `project` (I now use `PACKAGE_VERSION` there).